### ISSUE TITLE
Can we run Apollo on a designated 

### ISSUE THREAD

**Issue by xdzhang-xdu** [ApolloAuto/apollo#14738]
We appreciate you go through Apollo documentations and search previous issues before creating an new one. If neither of the sources helped you with your issues, please report the issue using the following form. Please note missing info can delay the response time.

### System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 18.04)**:
- **Apollo installed from (source or binary)**:
- **Apollo version (3.5, 5.0, 5.5, 6.0)**:
- **Output of `apollo.sh config` if on `master` branch**:

### Steps to reproduce the issue:
- **Please use bullet points and include as much details as possible**:

**Comment by daohu527** [comment]
Yes, apollo perception module can use gpu_id to select which gpu to use. then you can run simulator and apollo on different GPU

**Comment by xdzhang-xdu** [comment]
> Yes, apollo perception module can use gpu_id to select which gpu to use. then you can run simulator and apollo on different GPU

Thanks.
Where to find the usage doc?

**Comment by daohu527** [comment]
Simply put, apollo runs on gpu 0 by default, of course you can also modify the `gpu_id` in the configuration file in `modules/perception/pipeline/config`.

So I suggest that you can run lgsvl on gpu1, so you can save the configuration

**Comment by xdzhang-xdu** [comment]
> 

There are too many gpu_id configuration in Apollo project.
It is better to designating lgsvl on a specified GPU.
Do you know how to set?
